components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1343,7 +1347,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const uniqueURLs = new Set();
         return suggestions
             .sort((a, b) => {
-                // Most recently used first
+                // priority first
+                if (a.priority != b.priority) {
```

Comment:
I generally try to avoid "loose equality" (e.g. `"5" == 5`) -- is there any reason not to use `!==` here?

```suggestion
                if (a.priority !== b.priority) {
```